.babelrc

```diff
@@ -1,5 +1,8 @@
 {
   "optional": [
     "es7.objectRestSpread"
+  ],
+  "plugins": [
+    "babel-plugin-object-assign"
```

Comment:
Would we need this if we were using [`babel-runtime`](http://babeljs.io/docs/usage/runtime/)?